<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>UpArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Пережитки Яндекс.лицей</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>256</width>
      <height>91</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1) Базовые понятия, ветвления</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2) Цикл &quot;while&quot;/&quot;for&quot;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3) Всерос</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4) Контест</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5) Немного математики</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6) Личный кабинет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбор?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Try</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Немного  себе</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>e-mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>420</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>listWidget</zorder>
   <zorder>lineEdit</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
